### PR TITLE
Dump traceback to a temporary file.

### PR DESCRIPTION
**Pull request by wangxf123456**
The bug report will be dumped at "/tmp/django-deploy-bug-report-[random suffix]".

Also removed "try again" when internal errors happened because most of the time "try again" does not solve the problem.

### CODE REVIEW

File: django_cloud_deploy/crash_handling/__init__.py [hunk 1]
```diff
@@ -49,12 +50,22 @@ def handle_crash(err: Exception, command: str,
...
+        prefix='django-deploy-bug-report-')
     console.tell(
-        ('Your "{}" failed due to an internal error - please try again.'
+        ('Your "{}" failed due to an internal error.'
          '\n\n'
          'You can report this error by filing a bug on Github. If you agree,\n'
          'a browser window will open and an Github issue will be\n'
-         'pre-populated with the details of this crash.\n').format(command))
+         'pre-populated with the details of this crash.\n'
+         'See bug report at "{}" for more information.').format(
```
Comment (brianquinlan) [inline django_cloud_deploy/crash_handling/__init__.py]:
```suggestion
         'For more details, see: {}').format(
```

File: django_cloud_deploy/crash_handling/__init__.py [hunk 2]
```diff
@@ -49,12 +50,22 @@ def handle_crash(err: Exception, command: str,
...
+             command, log_file_path))
+    issue_content = _create_issue_body(command)
```
Comment (brianquinlan) [inline django_cloud_deploy/crash_handling/__init__.py]: I'd write the file before telling the user about it.

File: django_cloud_deploy/crash_handling/__init__.py (hunk 1 above)
Comment (wangxf123456) [reply django_cloud_deploy/crash_handling/__init__.py]: Done.

File: django_cloud_deploy/crash_handling/__init__.py (hunk 2 above)
Comment (wangxf123456) [reply django_cloud_deploy/crash_handling/__init__.py]: Done.

**Pull request merged**